api/proto/teleport/legacy/types/types.proto

```diff
@@ -7981,6 +7996,8 @@ message PluginStaticCredentialsOAuthClientSecret {
 // PluginStaticCredentialsSSHCertAuthorities contains the active SSH CAs used
 // for the integration or plugin.
 message PluginStaticCredentialsSSHCertAuthorities {
+  option (gogoproto.equal) = true;
```

Comment:
Instead of replying on the equal can we just consider compering the creds revision version ?

Reply:
> I also wouldn't like to rely on gogoproto features if that can be avoided. Maybe we can hand roll equality function?

I feel like this is overcomplicated. Sorry for confusion.  If a new creds will be added I'm sure that Equal function won't be extended. I have just asked if the revision can be used instead but right now it looks like we have Equal function that we need to maintenance.  I'm OK to use gogoproto.equal since this method is already used to comare plugins.